src/shell/Shell.cpp

```diff
@@ -412,6 +412,12 @@ PersistentRefHolder<ContextRef> createEscargotContext(VMInstanceRef* instance)
         {
             ObjectRef* dollor262Object = ObjectRef::create(state);
 
+            {
+                FunctionObjectRef::NativeFunctionInfo nativeFunctionInfo(AtomicStringRef::create(context, "gc"), builtinGc, 0, true, false);
+                FunctionObjectRef* buildFunctionObjectRef = FunctionObjectRef::create(state, nativeFunctionInfo);
+                dollor262Object->defineDataProperty(state, StringRef::createFromASCII("gc"), buildFunctionObjectRef, true, true, true);
```

Comment:
It seems that global `gc` function is already defined.
Does test262 need this function too?

Reply:
Yes. asyncGCDeref of ./test/test262/harness/async-gc.js needs $262.gc.
It is not documented, but we can see the usage

Reply:
Ok looks good :)